modules/serverless-upgrade-sub-channel.adoc

```diff
@@ -20,7 +20,7 @@ If you have selected Manual updates, you will need to complete additional steps
 . Navigate to the *Operators* → *Installed Operators* page.
 . Select the *{ServerlessOperatorName} Operator*.
 . Click *Subscription* → *Channel*.
-. In the *Change Subscription Update Channel* window, select *4.6*, and then click *Save*.
+. In the *Change Subscription Update Channel* window, select *4.7*, and then click *Save*.
```

Comment:
@alanfx @warrenvw same Q

Reply:
@abrennan89 Reverting this change as well.